src/I18n/Translator.php

```diff
@@ -174,6 +174,9 @@ protected function resolveContext($key, $message, array $vars)
         // No or missing context, fallback to the key/first message
         if ($context === null) {
             if (isset($message['_context'][''])) {
+                if ($message['_context'][''] === '') {
```

Comment:
Can you write this as

```php
if (context === null && isset($message['context'][''])) {
    return $message['_context'][''] === '' ? $key : $message['_context'][''];
}
```

It is starting to get confusing with all the nested `if` cases